cmd/kubelet/app/options/options.go

```diff
@@ -114,7 +114,7 @@ func NewKubeletServer() *KubeletServer {
 			RktStage1Image:                 "",
 			RootDirectory:                  defaultRootDir,
 			SerializeImagePulls:            true,
-			StreamingConnectionIdleTimeout: unversioned.Duration{5 * time.Minute},
+			StreamingConnectionIdleTimeout: unversioned.Duration{0},
```

Comment:
@ncdc I assume this is the bit you were asking about changing the default on. Can we make it really long before we make it completely unlimited?